src/feature/gui/gui-runner.c

```diff
@@ -540,12 +558,13 @@ void mGUIRun(struct mGUIRunner* runner, const char* path) {
 					if (showOSD) {
 						unsigned origin = runner->params.width - GUIFontHeight(runner->params.font) / 2;
 						unsigned w;
-						if (fastForward || (heldKeys & (1 << mGUI_INPUT_FAST_FORWARD_HELD))) {
+						bool fastForwarding = fastForward || (heldKeys & (1 << mGUI_INPUT_FAST_FORWARD_HELD));
+						if (fastForwarding) {
 							GUIFontDrawIcon(runner->params.font, origin, GUIFontHeight(runner->params.font) / 2, GUI_ALIGN_RIGHT, 0, 0x7FFFFFFF, GUI_ICON_STATUS_FAST_FORWARD);
 							GUIFontIconMetrics(runner->params.font, GUI_ICON_STATUS_FAST_FORWARD, &w, NULL);
 							origin -= w + GUIFontHeight(runner->params.font) / 2;
 						}
-						if (runner->core->opts.mute) {
+						if (runner->core->opts.mute || (fastForwarding && fastForwardMute)) {
```

Comment:
`runner->core->opts.mute` is the core's conception of if it's muted (as mentioned above) so the added clause should be redundant here.

Reply:
You're right, I removed the clause. Thanks!